### ISSUE TITLE
A checklist of things from the 12/16 version

### ISSUE THREAD

**Issue by Edaka** [LoneGazebo/Community-Patch-DLL#949]
You've seen these things on the forum - this is a reminder just in case one of them slip your mind.

1)Bromar1: "I've gotten a few sources of wheat on desert tiles. They had fresh water access from an oasis but I was unable to build farms on them. Is wheat supposed to be able to pop on desert tiles?"

2)Gidoza: "The +3 Culture for Garrisons in the Authority Tree is still giving -3 Culture and -1 Happiness per turn seemingly at city #3. Placing a garrison in the city simply zeroes the value." I'll just keep pushing it: perhaps the issue is no-EUI?

3)sleepyx732: "I've played Shoshone a fair bit, they [encampments] are supposed to cause 5 damage. Speaking of, 
    a) kasbah for Morroco does the same thing, which I don't believe is intended; 
    b) they can also be built next to each other, contrary to what the civilopedia says."

**Comment by Gidoza** [comment]
I'll add a couple more.

4.  Population bug:  A city can starve and 1 population will be lost, but not 1 citizen.  Doesn't go above 1, though.

5.  The +15 healing from God of War is not giving +15 healing.

**Comment by Edaka** [comment]
Those have already been looked at and fixed, I believe. They're among closed issues now. It's just that a version with the fix hasn't come out yet.

**Comment by LoneGazebo** [comment]
Indeed.

**Comment by LoneGazebo** [comment]
The civpedia elements don't really have a place - I'll mark them for later.
G

**Comment by Edaka** [comment]
I might as well throw this here. Regarding multiplayer - seems like overall it's very stable atm (cheers!), but it seems another desync culprit has been brought up now that people actually get a chance to get to the World Congress. Posted by Steelrat in the official forum MP modpack thread: 

"Hello, I've been playing with my friends version 12/16. Everything was good. Only one resync till world congress foundation. Then game host saw one proposal from congress host, everyone else one saw one same another. This happened for both AI propositions(we only went so far for now) and ofc next turn was resync. After the congress foundation resyncs became quite often. I'd say like one in 10 turns. If that has any difference: game host said that he set random personalities for AI - maybe thats a reason of different proposals."

tl;dr: WC resolutions proposed by the AI are displayed differently for game hosts and non-hosts.
